### DISCUSSION TITLE
Integration with datadog http://www.datadoghq.com/

### DISCUSSION THREAD

**Discussion by carlosaq** [Pierre-Lannoy/wp-decalog#42] [Integrations]
Does any have experience, reference or documentation about integrate the decalog observably information to datadog and generate it in the datadog dashboards to the monitoring and observability team. 
we have a critical site on wordpress and what to have events and alerts on our datadog monitoring room team.

**Comment by Pierre-Lannoy** [comment]
Hello @carlosaq ! ;)
